ipaserver/install/server/install.py

```diff
@@ -892,6 +892,10 @@ def install(installer):
     # Make sure the files we crated in /var/run are recreated at startup
     tasks.configure_tmpfiles()
 
+    if not options.no_pkinit:
+        print("Testing anonymous PKINIT")
+    krb.test_anonymous_pkinit()
```

Comment:
Testing PKINIT was added so that we fail early if the configuration is wrong. Otherwise you just get hard to debug issues such as login to WebUI failing suddenly. I can remove this if we are confident that the current code is robust enough.